DataParTemplate.m

```diff
@@ -87,6 +87,13 @@
 %                     - example for Philips 3D GRASE = '[1 2]' % (first control-label pair)
 %                     - example for Siemens 3D GRASE = 1 % first image
 %                     - example for GE 3D spiral = 2 % where first image is PWI & last = M0
+% x.DummyScanPositionInASL4D - indicates the position of Dummy scans in TimeSeries if they are integrated by the vendor in the 
```

Comment:
here the format of the parameter is missing (numerical, scalar or vector)

Reply:
Updated of M0Position. And added an example for null entry (as discussed with Maurice).

Reply:
What does "Updated of M0Position" mean? ;)

The null entry is fine if it is JSON and ExploreASL `[]` standard, I would only ensure that advanced parameters are not in the example JSON file.

Reply:
Haha - that's a fusion of "Update of M0Position" and "Updated M0Position" and I wanted to say that I have updated the description of the M0PositionInASL4D. It was late :) Comment sloppy, update itself not sloppy :)

Agree - DummyPositionInASL4D wasn't added to the example JSON.